Islandora/islandora
# Issue 2100 remove override facets js

**GitHub Issue**: (link)
https://github.com/Islandora/documentation/issues/2100

* Other Relevant Links (Google Groups discussion, related pull requests,
 Release pull requests, etc.)

# What does this Pull Request do?
This removes the override javascript that was causing problems executing the soft-limits.js code from the core facets module to address https://github.com/Islandora/documentation/issues/2100 

A brief description of what the intended result of the PR will be and/or what problem it solves.
the javascript libraries are no longer overridden - using the core facets/js/soft-limits and views-ajax javascript works as expected.

# How should this be tested?
To test, perform a search on a site that will give you at least 3 different facet values for a facet that uses a "List of links" configuration and has a value set for "Soft limit" ( The description for this "Limit the number of displayed facets via JavaScript." ) to limit facets at 3. 

When the code works correctly, there should be no console errors relating to that soft-limits.js. Also, the facet that was limited will now only display the top 3 facet values and a link to "Show more".
